src/verinfast/agent.py

```diff
@@ -453,7 +453,7 @@ def parseRepo(self, path: str, repo_name: str, branch: str = None):
             custom_args = [
                 "--config", "auto",
                 "--json",
-                f"--json-output={findings_file}",
+                f"--output={findings_file}",
                 "-q"
```

Comment:
For this see: https://semgrep.dev/docs/cli-reference. SG supports --json --output OR --json-output. For the version necessary for Python 3.9, --json-output was not supported yet.  It sill passes all unit tests for all Python versions with this syntax. And line 86 of test_yaml.py confirms findings are a valid JSON.